src/java.base/share/classes/sun/security/util/SecurityConstants.java

```diff
@@ -63,76 +58,13 @@ private SecurityConstants () {
 
     // Permission constants used in the various checkPermission() calls in JDK.
 
-    // java.lang.Class, java.lang.SecurityManager, java.lang.System,
-    // java.net.URLConnection, java.security.AllPermission, java.security.Policy,
-    // sun.security.provider.PolicyFile
+    // java.lang.Class, java.net.URLConnection, java.security.AllPermission
```

Comment:
BTW: It's okay to change the usage in Class.Holder.<clinit> to use `new AllPermission()`. That will help get to the point where this file can be removed.

Reply:
Fixed in https://github.com/openjdk/jdk/pull/22418/commits/5ad5e8c8b13e3aa8f8379def6b3bb3e994fd084d.